admin/src/main/java/io/strimzi/models/KafkaTopicDescription.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.models;
+
+import org.apache.kafka.clients.admin.TopicDescription;
+
+public class KafkaTopicDescription {
```

Comment:
You could keep the `record` here and just add the static method -> I think it is possible and you are not breaking anything.